list.txt

```diff
@@ -32213,6 +32641,7 @@ yahomail.gdn
 yahomail.top
 yahoo-mail.ga
 yahoo.co.au
+yahoo.com
```

Comment:
I cannot accept your PR, to many changes and this is another example of error. Sorry, you broke my trust :s...